### DISCUSSION TITLE
How to export the yolov5 model to onnx?

### DISCUSSION THREAD

**Discussion by stopthinking102** [airctic/icevision#1045] [Q&A]
Using icevision yolo with fastai. How to export it to onnx format?

**Comment by ai-fast-track** [comment]
You will need to save your trained model. Then replace `yolov5s.pt` by your model name that you just saved, and the image size you used during training:

```
python models/export.py --weights yolov5s.pt --img 640 --batch 1  # export at 640x640 with batch size 1
```
 The `export.py` is located here: https://github.com/ultralytics/yolov5/blob/master/export.py
 
 So, you will need to clone the `yolov5` repo on your local machine

For more info, check out the guide here below:
https://docs.ultralytics.com/tutorials/torchscript-onnx-coreml-export/

Please, ask your follow-up question in our forum
IceVision Forum: discord.gg/JDBeZYK
